### PR TITLE
Linked the Terms & Condition lable to the Terms & Condition Page

### PR DESCRIPTION
**Pull request by chithraka99**
In register page.tsx file, shows lots of changes because when I save the file it automatically formatted.

### CODE REVIEW

File: app/register/page.tsx [hunk 1]
```diff
@@ -53,16 +53,17 @@ const Register: React.FC = () => {
   const PostApiData = async () => {
     try {
       const result = await postRegsiterData(
-        "https://slc-open-hms-api.azurewebsites.net/register",formData)
-        .then((response) => {
+        "https://slc-open-hms-api.azurewebsites.net/register",
```
Comment (iamarasekera) [inline app/register/page.tsx:56]: Pass this url also from the data file.

File: app/register/page.tsx [hunk 2]
```diff
@@ -53,16 +53,17 @@ const Register: React.FC = () => {
...
         console.log(response);
         if (!response.ok) {
           console.log("Post Request Unsucceeful", response);
           setRegistrationMessage(response.title);
           setErrorMessages(Object.values(response.errors));
-        } else  {        
-           console.log("Post Request Successfull", response);
-          setRegistrationMessage('Registration successful!');
+        } else {
+          console.log("Post Request Successfull", response);
```
Comment (iamarasekera) [inline app/register/page.tsx:65]: Remove the console logs. Avoid committing console logs.

File: app/register/page.tsx [hunk 3]
```diff
@@ -296,14 +343,14 @@ const Register: React.FC = () => {
         </Grid>
       </form>
       {errorMessages.length > 0 && (
-              <ul>
-                {errorMessages.map((errorMessage, index) => (
```
Comment (iamarasekera) [inline app/register/page.tsx:300]: Don't use pure HTML elements. Please use List and ListItem from MUI

Comment (chithraka99) [reply app/register/page.tsx:300]: This part is a backend part. Is it okay to Update?

**Review by iamarasekera — APPROVED** [review]

**Pull request merged**